lab_3_ann_retriever/main.py

```diff
@@ -149,6 +183,8 @@ def __init__(self, corpus: list[list[str]]) -> None:
         Args:
             corpus (list[list[str]]): Tokenized documents to vectorize
         """
+        self._corpus = corpus
+        self.build()
```

Comment:
I do not think that you should call `build` there

Reply:
done